src/views/Dashboard.vue

```diff
@@ -95,6 +95,27 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
               You are not running Cylc UI via Cylc Hub.
             </v-tooltip>
           </div>
+          <div>
+            <v-list-item
+              id="jupyter-lab-button"
+              :disabled="!user.extensions?.lab"
+              :href="user.extensions?.lab"
+              target="_blank"
+            >
+              <template v-slot:prepend>
+                <v-icon size="1.6em">{{ $options.icons.jupyterLogo }}</v-icon>
+              </template>
+              <v-list-item-title class="text-h6 font-weight-light">
+                Jupyter Lab
+              </v-list-item-title>
+              <v-list-item-subtitle>
+                Open Jupyter Lab in a new browser tab.
+              </v-list-item-subtitle>
+            </v-list-item>
+            <v-tooltip :disabled="user.extensions?.lab">
+              Jupyter Lab is not installed.
```

Comment:
There's no way of telling without modifying the UIS PR providing this data whether `?.lab` is missing because it's not installed, or becuase `user` hasn't been endowed with that data is there?